GoRogue/MapGeneration/PolygonArea.cs

```diff
@@ -130,7 +184,115 @@ private void SetInnerPoints()
                 }
             }
         }
+        #endregion
+
+        #region static creation methods
+        /// <summary>
+        /// Creates a new Region from a GoRogue.Rectangle.
+        /// </summary>
+        /// <param name="r">The rectangle</param>
+        /// <param name="algorithm">Line-drawing algorithm to use for finding boundaries.</param>
```

Comment:
It may be useful to mention the constraints on the line-drawing algorithm here; ie. that it must be ordered (same as I think the `PolygonArea` constructor does).